builtin/logical/transit/path_keys.go

```diff
@@ -73,6 +73,8 @@ func pathPolicyRead(
 			"derived":                p.Derived,
 			"deletion_allowed":       p.DeletionAllowed,
 			"min_decryption_version": p.MinDecryptionVersion,
+			"latest_version":         p.LatestVersion,
+			"archive_version":        p.ArchiveVersion,
```

Comment:
If this is always the same as latest_version, there may be no point in exposing it